src/Microsoft.DotNet.Build.Tasks/PackageFiles/sign.targets

```diff
@@ -20,6 +20,11 @@
 
     <DelaySign>true</DelaySign>
     <DelaySign Condition="'$(UseOpenKey)' == 'true'">false</DelaySign>
+    <DelaySign Condition="'$(OSEnvironment)' == 'Windows_NT' And '$(IsTestProject)' == 'true'">false</DelaySign>
```

Comment:
Please add a comment about why Windows_NT is necessary and reference the roslyn issue tracking it.